<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Restaurante</class>
 <widget class="QMainWindow" name="Restaurante">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restaurante</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>611</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cardápio</string>
     </attribute>
     <widget class="QSplitter" name="splitter_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>521</width>
        <height>221</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nome_paciente"/>
      </widget>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Comida</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>porção do alimento (g)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bebida</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>porção da bebida (ml)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QComboBox" name="cb_comida">
          <item>
           <property name="text">
            <string>lasanha</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>arroz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>feijoada</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>batata</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>carne</string>
           </property>
          </item>
         </widget>
         <widget class="QComboBox" name="cb_quantidade_comida">
          <item>
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>450</string>
           </property>
          </item>
         </widget>
         <widget class="QComboBox" name="cb_bebida">
          <item>
           <property name="text">
            <string>suco</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>refrigerante</string>
           </property>
          </item>
         </widget>
         <widget class="QComboBox" name="cb_quantidade_bebida">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <item>
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>350</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500</string>
           </property>
          </item>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="btn_enviar">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>enviar</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cadastros</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tbl_dados">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Nome </string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bebida</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Comida</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Calorias</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar_2"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_in">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
  <action name="actionCarregar_2">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
